### DISCUSSION TITLE
GitHub Pages Does Not Render The Site Properly

### DISCUSSION THREAD

**Discussion by igotQweston** [thuliteio/doks#1121] [Q&A] [answered]
The site is default install using `npm create hyas@latest -- --template doks`

The only few changes I made are for like `baseurl` in config.toml (or now hugo.toml?).

Solutions that I found from other discussions didn't work for me. 

Here is my repo:
[https://github.com/igotQweston/igotQweston.github.io](https://github.com/igotQweston/igotQweston.github.io)

**Comment by h-enk** [comment, marked as answer]
Probably related.

There's an issue with [hugo_stats.json](https://github.com/gethyas/create-hyas/pull/31#issuecomment-1788148035). 

You'll need to run a local build and push the generated `hugo_stats.json` to GitHub.

Let me know if that helps

**Comment by Drilio** [comment]
Works for me for me here : https://gitlab.tetras-libre.fr/antoine-perso/doks-test 
Thanks !

**Comment by jollydodo** [comment]
Is there a fix for this already? My local builds look perfectly fine. If i clone my repository and run npm install in a clean environment everything works fine as well locally. However, the netlify deploys have a messed up layout (https://tiny-kitsune-5925a0.netlify.app/nl). I

**Reply by Drilio** [reply]
For some reasons it doesn't work for me anymore, didn't had time to inspect it further this week..

**Reply by h-enk** [reply]
> You'll need to run a local build and push the generated `hugo_stats.json` to GitHub.

Share your repository?

**Reply by Drilio** [reply]
there is the link  :https://gitlab.tetras-libre.fr/antoine-perso/doks-test

**Reply by h-enk** [reply]
> there is the link  :https://gitlab.tetras-libre.fr/antoine-perso/doks-test

The repo is private – could you (temporarily) make it public?

**Reply by Drilio** [reply]
I created a public github repo there : https://github.com/Drilio/dock-test

**Reply by h-enk** [reply]
@Drilio, I only changed the baseURL — https://github.com/h-enk/doks-1121/blob/main/config/_default/hugo.toml#L1

No issue (apart from missing content)? 

- [GitHub repository](https://github.com/h-enk/doks-1121)
- [Netlify deploy](https://doks-1121.netlify.app/)

**Reply by h-enk** [reply]
Here's an example deployment  on Gitlab Pages:

- [Gitlab repository](https://gitlab.com/h-enk/doks-gitlab-pages)
- [`.gitlab-ci.yml`](https://gitlab.com/h-enk/doks-gitlab-pages/-/blob/main/.gitlab-ci.yml?ref_type=heads)
- [Gitlab pages deploy](https://doks-gitlab-pages-h-enk-7f8afcd083c32d5a189605c5aab5a86eda737b2.gitlab.io/)

**Reply by Drilio** [reply]
The URL must have change because of my merge request... sorry ! And thank you for the help !
